src/server/node.rs

```diff
@@ -329,10 +329,10 @@ where
     {
         info!("start raft store thread"; "store_id" => store_id);
 
-        if self.store_handle.is_some() {
```

Comment:
Could we remove the `store_handle` field? I didn't find it is used elsewhere.

Reply:
Yes, it is removed, and replaced with `has_started`.